pyproject.toml

```diff
@@ -46,7 +46,8 @@ docs = [
     "scipy",
     "cocotb>=1.9.1",
 ]
-test = ["numpy>=1.25", "pytest>=8.2", "cocotb>=1.9.1"]
+test = ["numpy>=1.25", "pytest>=8.2"]
+test_hdl = ["numpy>=1.25", "pytest>=8.2", "cocotb>=1.9.1"]
```

Comment:
This should probably be used when installing the HDL-test, but can be changed later.